src/grammar.coffee

```diff
@@ -818,6 +819,7 @@ grammar =
     o '+     Expression',                      (-> new Op '+', $2), prec: 'UNARY_MATH'
 
     o 'AWAIT Expression',                       -> new Op $1 , $2
+    o 'AWAIT INDENT Object OUTDENT',            -> new Op $1 , $3
```

Comment:
Okay this is a total nit, but why is there a space before this comma? And on the previous line.

Reply:
@GeoffreyBooth guessing the previous line was originally copied from eg the `UNARY Expression` line, which contains an extra space to align with following. Updated